.github/workflows/test.yml

```diff
@@ -30,7 +30,7 @@ jobs:
           echo "::set-output name=dir::$(pip cache dir)"
 
       - name: Cache
-        uses: actions/cache@v2
+        uses: actions/cache@v3
```

Comment:
Can also bump codecov-action below:

```diff
-        uses: codecov/codecov-action@v1
+        uses: codecov/codecov-action@v3
```

Reply:
Done in https://github.com/xolox/python-humanfriendly/pull/69/commits/0ce390f2805c4d6506890eb7b1cbc8f28357a6bc